unreleased/add-link-attr-processor-doc.yaml

```diff
@@ -0,0 +1,11 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: resourceprocessor
+
+# A brief description of the change
+note: Add a link to the Attributes Processor `README` from the Resource Processor `README`
+
+# One or more tracking issues related to the change
+issues: [12077]
```

Comment:
There is no any user facing change. Can you please remove the changelog entry? I'll add a tag to skip the CI check

Reply:
I created an issue to discuss it. #12087